packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/create-model-customization-job.ts

```diff
@@ -0,0 +1,497 @@
+import { Construct } from 'constructs';
+import * as bedrock from '../../../aws-bedrock';
+import * as ec2 from '../../../aws-ec2';
+import * as iam from '../../../aws-iam';
+import * as kms from '../../../aws-kms';
+import * as s3 from '../../../aws-s3';
+import * as sfn from '../../../aws-stepfunctions';
+import { Stack, Token } from '../../../core';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/**
+ * The customization type.
+ *
+ * @see https://docs.aws.amazon.com/bedrock/latest/userguide/custom-models.html
+ */
+export enum CustomizationType {
+  /**
+   * Fine-tuning
+   *
+   * Provide labeled data in order to train a model to improve performance on specific tasks.
+   */
+  FINE_TUNING = 'FINE_TUNING',
+
+  /**
+   * Continued pre-training
+   *
+   * Provide unlabeled data to pre-train a foundation model by familiarizing it with certain types of inputs.
+   */
+  CONTINUED_PRE_TRAINING = 'CONTINUED_PRE_TRAINING',
+}
+
+/**
+ * The key/value pair for a tag.
+ */
+export interface ITag {
+  /**
+   * Key for the tag.
+   */
+  readonly key: string;
+
+  /**
+   * Value for the tag.
+   */
+  readonly value: string;
+}
+
+/**
+ * S3 bucket configuration for data storage destination.
+ */
+export interface BucketConfiguration {
+  /**
+   * The S3 bucket.
+   */
+  readonly bucket: s3.IBucket;
+  /**
+   * The prefix for the S3 bucket.
+   *
+   * @default - no prefix
+   */
+  readonly prefix?: string;
+}
+
+/**
+ * VPC configuration
+ */
+export interface IBedrockCreateModelCustomizationJobVpcConfig {
+  /**
+   * VPC configuration security groups
+   *
+   * The maximum number of security groups is 5.
```

Comment:
```suggestion
   * The minimum number of security groups is 1.
   * The maximum number of security groups is 5.
```
Let's specify both boundaries.